style-guide/google-format-ci-v0.1.sh

```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env bash
-
+set -eu
 ###################################################################################################
  # This script downloads google formatter and displays formatting issues on Github actions
 ###################################################################################################
-
-curl -LJO "https://github.com/google/google-java-format/releases/download/v$GOOGLE_JAR_VERSION/$GOOGLE_JAR_NAME"
+GOOGLE_JAR_VERSION=${GOOGLE_JAR_VERSION:-"1.11.0"}
```

Comment:
So I can run it over all the code locally intead of using commit hook ... which is not working for me for some reason

Reply:
You might need to install Precommit on your local system .e.g, `brew install pre-commit`

Reply:
yeah I have installed pre-commit as in the docs but it still does not do  the pre-commit hooks :/